common/player.go

```diff
@@ -4,7 +4,7 @@ import (
 	"time"
 
 	"github.com/golang/geo/r3"
-
+	
```

Comment:
now it has an additional tab (I'm very sorry, I have some sort of whitespace OCD) 😄

Reply:
Haha it's ok Richard 😄 ! 
Done